internal/guest/bridge/bridge_v2.go

```diff
@@ -133,16 +136,41 @@ func (b *Bridge) createContainerV2(r *Request) (_ RequestResponse, err error) {
 //
 // This is allowed only for protocol version 4+, schema version 2.1+
 func (b *Bridge) startContainerV2(r *Request) (_ RequestResponse, err error) {
-	_, span := trace.StartSpan(r.Context, "opengcs::bridge::startContainerV2")
+	var (
+		request prot.MessageBase
+		c       *hcsv2.Container
+		cg      cgroups.Cgroup
+	)
```

Comment:
I'd prefer to have the spans at the top of the function and then the vars closer to where they'll be used.